euth/dashboard/views.py

```diff
@@ -120,10 +120,16 @@ class DashboardProjectListView(DashboardBaseMixin,
     filter_set = filters.ArchivedFilter
 
     def get_queryset(self):
-        return super().get_queryset().filter(
+        qs = super().get_queryset().filter(
             organisation=self.organisation
         )
 
+        for project in qs:
```

Comment:
The querysets are lazy, by iterating through them here you are forcing the orm to load all project from the data base. Even if they would be filtered at a later stage (eg, by pagination or filters).

There are two other ways to archive the same. Either define is_archivable as a getter (property) on the project. Or use the [annotate](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#annotate) feature of the queryset.